### PR TITLE
MEN-4894 Monitoring: full integration test: log patterns alerting.

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline por favor

**Comment by merlin-northern** [comment]
I really have no idea why the pipeline was skipped: https://gitlab.com/Northern.tech/Mender/integration/-/pipelines/350232806

**Comment by lluiscampos** [comment]
> I really have no idea why the pipeline was skipped: https://gitlab.com/Northern.tech/Mender/integration/-/pipelines/350232806

That is the new way the bot triggers the individual pileine: first push the branch skipping the pipeline, then run manually the pipeline. If you pay attention you'll see that it all repositories

File: tests/tests/test_monitor_client.py [hunk 1]
```diff
@@ -215,6 +246,41 @@ def test_monitorclient_alert_email(self, monitor_commercial_setup_no_client):
...
+            "test_monitorclient_alert_email: email alert on log file containing a pattern scenario."
+        )
+        log_file = "/tmp/mylog.log"
+        mender_device.run("echo 'some line' >> " + log_file)
+        prepare_log_monitoring(
+            mender_device, service_name, log_file, "session opened for user [a-z]*",
+        )
+        time.sleep(2 * wait_for_alert_interval_s)
+        mender_device.run("echo 'some line' >> " + log_file)
+        time.sleep(wait_for_alert_interval_s)
```
Comment (lluiscampos) [inline tests/tests/test_monitor_client.py:259]: Around here we should `assert` that no e-mail has been received (iow the pattern has not yet been found)

File: tests/tests/test_monitor_client.py [hunk 2]
```diff
@@ -163,6 +191,9 @@ def test_monitorclient_alert_email(self, monitor_commercial_setup_no_client):
         )
         logger.info("test_monitorclient_alert_email: env ready.")
 
+        logger.info(
+            "test_monitorclient_alert_email: email alert on systemctl service not running scenario."
```
Comment (lluiscampos) [inline tests/tests/test_monitor_client.py:195]:
```suggestion
            "test_monitorclient_alert_email: email alert on systemd service not running scenario."
```

**Review by lluiscampos — APPROVED** [review]
Looks good :+1:

File: tests/tests/test_monitor_client.py (hunk 2 above)
Comment (merlin-northern) [reply tests/tests/test_monitor_client.py]: fixed

File: tests/tests/test_monitor_client.py (hunk 1 above)
Comment (merlin-northern) [reply tests/tests/test_monitor_client.py:260]: added!

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline por favor

**Comment by merlin-northern** [comment]
> > I really have no idea why the pipeline was skipped: https://gitlab.com/Northern.tech/Mender/integration/-/pipelines/350232806
> 
> That is the new way the bot triggers the individual pileine: first push the branch skipping the pipeline, then run manually the pipeline. If you pay attention you'll see that it all repositories

I realize that, however I got the impression that this time the pipeline was not started at all. ok, thanks.

**Comment by merlin-northern** [comment]
fix as per review comments, the failure in the rbac test is related to this: https://github.com/mendersoftware/integration/pull/1517 the present test passes, merging this one, as it does not conflict with pull/1517

**Pull request merged**